ports/voidstranger/port.json

```diff
@@ -0,0 +1,30 @@
+{
+  "version": 3,
+  "name": "voidstranger.zip",
+  "items": [
+    "Void Stranger.sh",
+    "voidstranger/"
+  ],
+  "items_opt": null,
+  "attr": {
+    "title": "Void Stranger",
+    "porter": [
+      "nate_____", "gooeyPhantasm"
+    ],
+    "desc": "Void Stranger is a 2D sokoban-style puzzle game where every step counts. Descend into the forgotten labyrinth teeming with fiendish foes and traps that defy reason. A swift defeat lurks at every corner for those who neither study their surroundings nor think their moves through with care.\nTrust you wits and slowly, surely, you will conquer the mysteries before you.\n\nYou do remember why you are here, right? Or have you lost something? Something very important to you.",
+    "desc_md": null,
```

Comment:
Same goes for description. On device more short description and _md for full markdown on website. Space on device details is rather limited.